consume-more-stuff/server/routes/categories.js

```diff
@@ -1,8 +1,129 @@
 const express = require("express");
 const router = express.Router();
+const Category = require("../db/Models/Category");
+const Item = require("../db/Models/Item");
 
 router.get("/", (req, res) => {
-  res.send(`smoke test - categories`);
+  return Category.fetchAll()
+    .then(cats => {
+      res.json(cats);
```

Comment:
Good habit to get into `return` in front of all `res.*` responses.

Reply:
added return on line 9